src/constants/type-info.js

```diff
@@ -271,7 +271,43 @@ export default {
     icon: Icons.Table,
     adapter: LocalAdapters[types.TABLE],
     mustGroup: false,
-    uniqueOptions: {},
+    uniqueOptions: {
+      hidePagination: {
+        name: 'Hide Pagination',
+        type: Boolean,
+        defaultValue: false,
+      },
+      headerTitle: {
+        name: 'Header Title',
+        type: Boolean,
+        defaultValue: false,
+      },
+      compactTable: {
+        name: 'Compact Table',
+        type: Boolean,
+        defaultValue: false,
+      },
+      centerHeader: {
+        name: 'Center Header',
```

Comment:
Change Name to`Centered Header`. We tend not to use verbs in these names. I just change it to keep consistency.